core/src/commonMain/kotlin/com/ustadmobile/core/controller/VideoContentPresenterCommon.kt

```diff
@@ -1,6 +1,8 @@
 package com.ustadmobile.core.controller
 
 import com.ustadmobile.core.account.UstadAccountManager
+import com.ustadmobile.core.contentformats.xapi.Statement
```

Comment:
This class still has the handleUpNavigation() function. Can this be removed as we now use jetpack navigation?